scripts/check_deps.sh

```diff
@@ -4,3 +4,4 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
+go mod verify
```

Comment:
I can move this two commits earlier.

Reply:
My understanding is this verifies the cache. AFAICT, there's still no "good" way to do this with modules. [1]

1: https://github.com/golang/go/issues/27348#issuecomment-573117775

Reply:
as opposed to checking (should it exist) the vendor directory?

Would `go mod tidy` help at all? 

I'm unsure what the difference here is specifically between `dep check` and this.